Feature/7: Footer

YumeAyasaki opened the pull request:
In this PR, I have:

- Checkout from the feature/8 branch...
- Make a footer component...
- Add icons to the public folder...
- Include said component to the _app.js file, thus making it a theme component...

Note: About the icon, should we use a library for that, or use the font in Figma or just export and include it inside the public folder...?
And the linking thing is not yet available...

![localhost_3000_Test - Cá nhân 1 - Microsoft​ Edge 9_16_2022 11_54_37 AM](https://user-images.githubusercontent.com/43096050/190559604-eea9f6f5-872c-46b3-a953-540cb5baed43.png)

YumeAyasaki committed:
File create and minor edit

Create file for mixins and variables, rename globals and home (module) to scss and some minor edit in index.js and _app.js...

YumeAyasaki committed:
Color variables

Add color variables + include variables file and mixin file in global.scss...

YumeAyasaki committed:
Global styling

Add global styling (header, body, small, details) + Make page test (just for general global styling, not variables yet)...

YumeAyasaki committed:
Global styling(1)

Minor edit for test page...

YumeAyasaki committed:
Initialize footer

Module scss and schema(?) for footer

YumeAyasaki committed:
Position + title

Mostly working on component's position + basic title (?)...

YumeAyasaki committed:
Edit text, add icon

Add styling for text, facebook and youtube icon and media queries (phone) for the main text. Note: Add more about the media queries...

YumeAyasaki committed:
Add color + Edit text font

Add accent light and rename Poppins to Lora...

YumeAyasaki committed:
Delete not-relevant file

Delete Header file...

YumeAyasaki committed:
Global styling

Copy global styling from feature/8...

YumeAyasaki committed:
Main text editing

Change from justify to center...

YumeAyasaki committed:
Include footer in _app.js

Make footer a theme component...

PaulHo0501 commented on components/theme/footer.jsx:
```diff
@@ -0,0 +1,32 @@
+    <div className={styles.footer}>
+      {/* Title */}
+      <h5 className={styles.title}> 
+        Vườn hoa sứ
```
Change this to "Vườn Hoa Sứ"

PaulHo0501 commented on components/theme/footer.jsx:
```diff
@@ -0,0 +1,32 @@
+      
+      {/* Icon */}
+      <div className={styles.iconContainer}>
+        <div className={styles.icon}><Image src='/icons/facebook.svg' alt='facebook' layout='fill' objectFit='contain'/></div>
```
Please use the library "[react-icons](https://react-icons.github.io/react-icons/)" for this

PaulHo0501 commented on components/theme/footer.module.scss:
```diff
@@ -0,0 +1,45 @@
+    
+    color: white;
+
+    .title {
```
Apply global styling to these element after changing the styling into classes or mixins

PaulHo0501 reviewed (changes requested):
Please check the comments.

YumeAyasaki committed:
Rename + react-icons

Rename Vuon Hoa Su and add react-icons...

YumeAyasaki committed:
Mixins and classes

Add mixins for Lora font and change all elements (tag?) to class...

YumeAyasaki committed:
Edit test page

Note: Body, smallest and details are not include...

YumeAyasaki committed:
Finalize global styling

Body, smallest and details. Editing test page for future test and other things...

YumeAyasaki committed:
Merge branch 'feature/8' of https://github.com/MEOStudio/vuonhoasu.org into feature/7

YumeAyasaki committed:
Adjusting with global styling

Adjust small thing to make it compatible with new global styling...

YumeAyasaki commented:
Update PR:
- Rename to Vuon Hoa Su...
- Apply react-icons...
- Edit minor things to match the new global styling...

Note: As you can see, the old module sass file doesn't match well with the new global styling. Any idea for that...?

PaulHo0501 commented on components/theme/footer.module.scss:
```diff
@@ -31,15 +31,15 @@
     }
```
Please capitalize the component's name (eg Footer, not footer)

YumeAyasaki committed:
Rename camelUpperCase

Rename class + minor edit for test page...

YumeAyasaki committed:
Rename camelUpperCase (1)

Rename class + minor edit for test page...

YumeAyasaki committed:
Merge branch 'feature/8' of https://github.com/MEOStudio/vuonhoasu.org into feature/7

YumeAyasaki committed:
Change name file + class

Name file (component) to first-letter-uppercase and class name to the new global styling...

YumeAyasaki committed:
Rename footer.jsx to Footer.jsx

For some reason, changing the file's name in VS Code doesn't apply out here. This's a manual commit, make on github page...

YumeAyasaki committed:
Rename footer.module.scss to Footer.module.scss

The same as the below commit, manual change on github page...

YumeAyasaki commented:
Update PR:
- Update some name classes...
- Rename footer to Footer...

Note: Do you have any idea regarding the name change commit of VS Code...?

PaulHo0501 commented:
LGTM
